easy_phi/app.py

```diff
@@ -123,16 +157,26 @@ def get(self):
         return response
 
 # URL schemas to RequestHandler classes mapping
-APPLICATION = tornado.web.Application([
-    (r"/", VersionHandler),  # Default page
-    (r"/version", VersionHandler),
-    (r"/modules", ModulesListHandler),
-    (r"/pick_module", SelectModuleHandler),
-    (r"/send_command", SCPICommandHandler),
-    (r"/admin", AdminConsoleHandler)
-])
+application = tornado.web.Application([
+    (r"/", tornado.web.RedirectHandler,
+        {"url": '/static/index.html'}),
+    (r"/api/v1/info", PlatformInfoHandler),
+    (r"/api/v1/modules", ModulesListHandler),
+    (r"/api/v1/module", SelectModuleHandler),
+    (r"/api/v1/scpi", SCPICommandHandler),
```

Comment:
I agree with @rezniknn that at this point we don't have time to pass all design decisions through official document. 
Now, about design decisions. GET requests should not change anything in the system, and you effectively propose to set status (selected/not selected) by GET request. Also, for user selection we need both GET and POST to read and update status.
As for SCPI commands, we need to get module list of available commands (aka configuration), which does not change module status, and send SCPI commands which may or may not change something in the system. So, we need both GET and PUT  there as well.
Also, please note that user lock and SCPI concepts are semantically two different entities, I would prefer not to mix them in one handler